doc: peripheral_hids_keyboard,mouse encryption notice (#6454)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth_services/services/hids.rst

disconnect events using the dedicated API. This is done to synchronize
the connection state of HIDS with the top module that uses it.

.. note::

>    Certain systems require encryption to be enabled in order to connect to a device.
>    Thus :kconfig:`BT_HIDS_DEFAULT_PERM_RW_ENCRYPT` should be enabled to satisfy this requirement.

Reviewer: ```suggestion
   In some systems, you need to enable encryption to be able to connect to a device.
   To satisfy this requirement, enable the :kconfig:`CONFIG_BT_HIDS_DEFAULT_PERM_RW_ENCRYPT` option.
```
Should we specify what kind of systems require the encryption?
